lib/elephant_in_the_room/sites/sites.ex

```diff
@@ -18,7 +18,9 @@ defmodule ElephantInTheRoom.Sites do
 
   """
   def list_sites do
-    Site |> Repo.all()
+    Site
+    |> Repo.all()
+    |> Repo.preload([:categories, :posts, :tags])
```

Comment:
It may be problematic to preload all the posts when a site has a lot of then. Is there any way to limit that?